DOCSP-47709 - move connection pool options (#178)
mongodb/docs-pymongo: source/connect/connection-options.txt

       | **MongoClient Example**: ``server_selector = your_function``
       | **Connection URI Example**: N/A

Connection Pools
~~~~~~~~~~~~~~~~

A **connection pool** is a cache of open database connections maintained by {+driver-short+}.

> When your application requests a connection to MongoDB, {+driver-short+} seamlessly

Reviewer: S: consider removing superfluous adverb
```suggestion
When your application requests a connection to MongoDB, {+driver-short+}
```

Author: agreed

gets a connection from the pool, performs operations, and returns the connection

> to the pool for reuse. Connection pools help reduce application latency and the number
> of times new connections are created by {+driver-short+}.

Reviewer: ```suggestion
to the pool for reuse. Connection pools help reduce application latency and the number
of times that {+driver-short+} must create new connections.
```

Author: agreed
